calico/_includes/release-notes/v3.23.0-release-notes.md

```diff
@@ -0,0 +1,145 @@
+09 May 2022
+
+#### IPv6 VXLAN support
+
+Calico now supports VXLAN encapsulation for IPv6 networks.
+
+Pull Requests:
+
+ - Add IPv6 VXLAN support [calico #5988](https://github.com/projectcalico/calico/pull/5988) (@coutinhop)
+ - Add IPv6 VXLAN route calculation support to Felix [calico #5919](https://github.com/projectcalico/calico/pull/5919) (@coutinhop)
+ - Allocate IPv6 VXLAN tunnel addresses [calico #5893](https://github.com/projectcalico/calico/pull/5893) (@coutinhop)
+
+#### VPP dataplane beta
+
+The Calico VPP dataplane integration is now in beta!
+
+#### Calico networking support in AKS
+
+You can now use the Calico CNI in your AKS clusters to take advantage of all of the Calico networking features.
+
+Pull Requests:
+
+ - Updated Calico doc for AKS Bring Your Own CNI. [calico #5857](https://github.com/projectcalico/calico/pull/5857) (@song-jiang)
+
+#### BGP enhancements
+
+We have added additional configuration options to allow for more control in specific scenarios, such as node-to-node mesh and separate systems running Calico using the same AS.
+
+Pull Requests:
+
+ - Add BGP configuration options for turning off BGP loop prevention for eBGP peers. [calico #5736](https://github.com/projectcalico/calico/pull/5736) (@mgleung)
+ - Add passive configuration for global BGP peers [calico #5625](https://github.com/projectcalico/calico/pull/5625) (@mgleung)
+ - Add maxRestartTime and BGP password fields to BGP configurations for full mesh configurations. [calico #5609](https://github.com/projectcalico/calico/pull/5609) (@mgleung)
+
+#### Container Storage Interface (CSI) support
+
+Following the Kubernetes move from FlexVolumes, Calico now has its own CSI driver. Features that used to depend on FlexVolumes have been migrated to instead use CSI.
```

Comment:
Probably want to call out the upgrade / install implications for ALP more specifically and in more detail here.